dashboard/src/scenes/report/view.js

```diff
@@ -174,7 +183,11 @@ const View = () => {
                 <ButtonCustom color="link" className="noprint" title="Suivant" disabled={reportIndex === 0} onClick={onNextReport} />
               </div>
             </div>
-            <div>{`Compte rendu du ${toFrenchDate(report.date)} de l'équipe ${currentTeam.name}`}</div>
+            <div>
+              {`Compte rendu de l'équipe ${currentTeam.name}`}
+              <br />
+              {getPeriodTitle(report.date, currentTeam.nightSession)}
```

Comment:
Voir au dessus (mais pareil ça a peut être pas de sens)

```suggestion
              {getPeriodTitle(report.date, currentTeam?.nightSession)}
```